task_schedule.pl

```diff
@@ -260,6 +262,21 @@
     # Exit if all the cron tasks have been disabled
     exit(0) unless grep { not $_->{disabled} } @crontab;
 
+    ##*****************************************************
+    ##  Check elapsed run time and exit if over threshold
+    ##  (minus a minute)
+    ##*****************************************************
+    my $curr_time = time();
+    my $run_time = $curr_time - $start_time;
```

Comment:
Sure, though adding the 60 seconds there does make it a bit of a fib (not actual run time) and is harder on my brain.